core/model/customlist.py

```diff
@@ -112,6 +112,15 @@ def __lt__(self, other):
             other.name,
         )
 
+    @classmethod
+    def list_entries(cls, _db: Session, list_id: int):
```

Comment:
Requesting one minor change to avoid confusion: can we rename this to `entries_having_works` or something along those lines to hint that this is possibly different from `self.entries`, which contains all entries in the list, regardless of whether or not they have works.